brayns/tasks/LoadDataFunctor.cpp

```diff
@@ -49,11 +49,11 @@ void LoadDataFunctor::operator()(Blob&& blob)
     {
         struct Scope
         {
-            Scope(Blob&& blob, LoadDataFunctor& parent)
+            Scope(Blob&& b, LoadDataFunctor& parent)
```

Comment:
why this change?

Reply:
Clang complains about variable shadowing the other one line 62. Gcc is happy though. I had this issue when compiling on MacOSX

Reply:
ah ok